ports/stm32/mboot/mboot_pack_dfu.py

```diff
@@ -228,11 +230,22 @@ def verify_pack_dfu(keys, filename):
                 chunk, 0, MBOOT_PACK_HYDRO_CONTEXT, keys.secretbox
             )
             assert chunk is not None
+            binary_data += chunk
```

Comment:
This needs to go after the decompress below.

Reply:
Moved after decompress